### PR TITLE
[Auto] Update vim docs

### PR DESCRIPTION
**Pull request by VimspectorUpdateBot**
Update the vim docs after recent changes

### CODE REVIEW

**Comment by puremourning** [comment]
This change is [Reviewable](https://reviewable.io/reviews/puremourning/vimspector/544)

**Review by puremourning — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/puremourning/vimspector/544#-:-MxQR9rf35hxszFkqG0O:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @VimspectorUpdateBot)

**Review by puremourning — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/puremourning/vimspector/544#-:-MxQSipm6qfiz3lW_fYo:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @VimspectorUpdateBot)

File: doc/vimspector.txt [hunk 1]
```diff
@@ -167,10 +169,10 @@ Image: Build [3] Image: Matrix [5] Image: Gitter [7]
 
 - Debug profile configuration
 
-  - C, C++, Rust, etc.
+  - C, C , Rust, etc.
```
Comment (mark2185) [inline doc/vimspector.txt:172]: _Gee Bill, two Cs?_

Comment (puremourning) [reply doc/vimspector.txt:172]: Ugh. I forgot I had to fix that up. I’ll do it next time.

**Pull request merged**